iGCS/Models/Craft/GCSCraftArduPlane.m

```diff
@@ -30,11 +36,34 @@ @implementation GCSCraftArduPlane
         _guidedMode  = APMPlaneGuided;
         _setModeBeforeGuidedItems = NO;
         _icon = [UIImage imageNamed:@"plane-icon-128.png"];
+        _notificationCenter = [NSNotificationCenter defaultCenter];
+
+        // set heartbeat to nil when app goes into background so all
+        // events fire again on app launch and telemetry reconnect
+        __weak GCSCraftArduPlane *weakSelf = self;
+        [self.notificationCenter addObserverForName:UIApplicationDidEnterBackgroundNotification
+                                             object:nil
+                                              queue:self.craftQueue
+                                         usingBlock:^(NSNotification *note) {
+                                             [weakSelf clearHearbeat];
+                                         }];
+
+        // set heartbeat to nil when app loses telemetry for a period of time
+        [self.notificationCenter addObserverForName:GCSHeartbeatManagerHeartbeatWasLost
+                                             object:nil
+                                              queue:self.craftQueue
+                                         usingBlock:^(NSNotification *note) {
+                                             [weakSelf clearHearbeat];
```

Comment:
Unclear we want to do this:
* when telemetry is re-established, will speak the various alerts (regardless of whether or not it changed since last heartbeat was received)
* potentially invalidates the values of isInAutoMode etc